Vagrantfile

```diff
@@ -29,6 +29,7 @@ Vagrant::configure("2") do |config|
   end
 
   config.vm.define "clean" do |vm|
```

Comment:
Is the `dokku` virtual machine unused now? If this is the case maybe it's a idea to remove it. Also maybe its useful to make the `clean` virtual machine the primary one.